modules/hdbt_admin_tools/assets/js/linkPluginEnhancements.js

```diff
@@ -5,7 +5,7 @@
   'use strict';
 
   // Simple show / hide functionality.
-  const handleVisibilty = function (element, show = true) {
+  const handleVisibility = function (element, show = true) {
```

Comment:
This could probably be written as:

```
  const handleVisibility = (element, show = true) => (
    (show) ? element.show(): element.hide()
  );
```